edn/src/edn.rustpeg

```diff
@@ -76,25 +76,22 @@ namespace_separator = "/"
 symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
 symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
 
-symbol_namespace = symbol_char_initial+ (namespace_divider symbol_char_subsequent+)*
+symbol_namespace = symbol_char_initial+ symbol_char_subsequent* (namespace_divider symbol_char_subsequent+)*
 symbol_name = ( symbol_char_initial+ / "." ) ( symbol_char_subsequent* / "." )
 
 keyword_prefix = ":"
 
-// TODO: More chars here?
-keyword_namespace_char = [a-z] / [A-Z] / [0-9]
-keyword_namespace = keyword_namespace_char+ (namespace_divider keyword_namespace_char+)*
-
-keyword_name_char = [a-z] / [A-Z] / [0-9] / "."
-keyword_name = keyword_name_char+
-
 pub symbol -> Value =
-    ns:( sns:$(symbol_namespace) namespace_separator { sns })? n:$(symbol_name) {
+    ns:( sns:$(symbol_namespace) namespace_separator {
+        sns
+    })? n:$(symbol_name) {
         types::to_symbol(ns, n)
     }
 
 pub keyword -> Value =
```

Comment:
I've read that, and the following stood out:

> Keywords follow the rules of symbols, except they can (and must) begin with :, e.g. :fred or :my/fred

It's the reason I unified those regexps. Am I misunderstanding the above? Should we not do this?